### DISCUSSION TITLE
Bundling part of the application as a service worker

### DISCUSSION THREAD

**Discussion by m-a-r-c-e-l-i-n-o** [vercel/next.js#10654] [Help]
Hello there,

We have a fairly complex application and we've seen drastic performance gains from including the entire Redux workflow inside a web worker, but that created some maintenance concerns. Is there a way to tell Next.js to build a portion of the app as a web worker? In other words, we need a "web worker bundle" and an "app bundle". 

We will most likely use: https://github.com/dai-shi/redux-in-worker

**Comment by lucacasonato** [comment]
> Bundling part of the application as a service worker

The title is wrong. It should be 'Bundling part of the application as a web worker'. Service workers != web workers
